.github/workflows/master.yml

```diff
@@ -46,7 +46,6 @@ jobs:
     steps:
     - uses: actions/checkout@v3
       with:
-        submodules: recursive
```

Comment:
Why is this line removed?

Reply:
I accidentally added in the MLP release PR. I don't think we should check lint for git submodules. So removed it in this PR